lib/test/unit/ui/console/testrunner.rb

```diff
@@ -454,6 +461,14 @@ def test_finished(test)
             end
             @already_outputted = false
 
+            if @options[:report_slow_tests]
+              @slow_tests << {
```

Comment:
How about renaming this to `@test_statistics` or something?
Because `@slow_tests` may include not slow tests.

Reply:
Sounds nice!